src/activitypub/mod.rs

```diff
@@ -1,4 +1,5 @@
 use db;
+use db::datetime::Rfc339able;
```

Comment:
maybe we should add another 3 to this module name. `db::datetime::Rfc3339able` strikes me as something slightly more recognisable and usable in a Datetime context than "MLTNET - A "MULTI-TELNET" SUBSYSTEM FOR TENEX"